CHANGES.rst

```diff
@@ -9,6 +9,12 @@ detailed information about all changes.
 
 Below is a summary:
 
+Version 0.3.1
```

Comment:
These changes should actually go under version 0.3, which is not yet released, which I realise the changelog doesn't make clear.